flake.nix

```
@@ -0,0 +1,36 @@
+{
+  description = "onefetch";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+  };
+
+  outputs = {
+    nixpkgs,
+    self,
```

```suggestion
    self,
    nixpkgs,
```
As I said I'm not too familiar, but based on the docs I'm seeing it seems standard to list `self` first. IDK if position matters here.